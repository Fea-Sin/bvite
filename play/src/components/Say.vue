<script lang="ts">
import { defineComponent } from "vue";

export default defineComponent({
  props: {
    name: {
      type: String,
      default: "Bvite",
    },
  },
});
</script>

<template>
  <div class="sayBox">
    {{ name }}
  </div>
</template>

<style lang="less" scoped>
.sayBox {
  padding: 30px;
  font-size: 30px;
  text-align: center;
  color: blueviolet;
}
</style>
