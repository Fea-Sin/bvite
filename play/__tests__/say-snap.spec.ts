import { shallowMount } from "@vue/test-utils";
import Say from "../src/components/Say.vue";

describe("Say Sanpshot", () => {
  it("renders correctly", () => {
    const wrapper = shallowMount(Say, {
      props: {
        name: "Hello Bvite",
      },
    });

    expect(wrapper.element).toMatchSnapshot();
  });
});
