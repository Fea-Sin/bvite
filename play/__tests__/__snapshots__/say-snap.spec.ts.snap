// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Say Sanpshot renders correctly 1`] = `
<div
  class="sayBox"
>
  Hello Bvite
</div>
`;
